doc/Caliptra.md

```diff
@@ -362,11 +385,37 @@ The SoC may support a fuse bank for representing the hash of the owner's public
 
 The owner key, when represented in fuses or in the FMC's alias certificate, is a SHA384 hash of a structure that contains a list of owner public keys. This supports key rotation.
 
+## Provisioning UDS during Manufacturing
+
+![](./images/Manuf-UDS-Flow.png)
+
+*Figure 6: Subsystem Mode: UDS manufacturing flow*
```

Comment:
Is there anything tying these flows to Subsystem? I think Subsystem and Passive can be the same.

Reply:
+1. Let's look into whether these manufacturing flows can/should be used in either mode. It would be nice to only have to maintain one set of flows.

Reply:
This is talking about how Caliptra is responsible for writing to fuse controller for programming UDS over the AXI manager interface. We wont have this interface in passive mode since we said Caliptra will only have AXI SUB in passive.

Reply:
But is that a beneficial way of designing the feature? the value could have just been written to a Caliptra register for example.

Reply:
I definitely don't want to do that. If folks want to use passive mode, let them adhere to 1.x mode of UDS seed generation. There is already lot of trust on the SOC manager. Don't want to add/validate/maintain these smaller configs.